sdks/python/apache_beam/runners/portability/flink_runner.py

```diff
@@ -32,18 +32,18 @@
 
 class FlinkRunner(portable_runner.PortableRunner):
   def default_job_server(self, options):
-    flink_master_url = options.view_as(FlinkRunnerOptions).flink_master_url
-    if flink_master_url == '[local]' or sys.version_info < (3, 6):
+    flink_master = options.view_as(FlinkRunnerOptions).flink_master
```

Comment:
Actually, I just realized that in Python this is different from Java. There a various inconsistencies, also in the job server. Pushing a commit to adjust this by allowing both arguments.

Reply:
Will do that in a follow-up.

Reply:
Thanks, let me know when you have a PR. I'd like to get this in by 2.17 so we can advertise this as a release thats really easy to use with Flink.

Reply:
https://github.com/apache/beam/pull/9844